libnavigation-ui/src/main/java/com/mapbox/navigation/ui/instruction/InstructionView.java

```diff
@@ -334,6 +381,33 @@ public void hideInstructionList() {
     onInstructionListVisibilityChanged(false);
   }
 
+  /////////////////////////////////////////
+  private void showGuidanceView(double distanceTraveled, double currentStepTotalDistance) {
+    int thresholdToShowGuidanceView = 200;
+    if (currentStepTotalDistance <= thresholdToShowGuidanceView) {
+      routeJunction = new RouteJunction(appendTokenTo(), currentStepTotalDistance);
+      animateShowGuidanceViewImage();
+      new Picasso.Builder(getContext()).build().load(routeJunction.getGuidanceImageUrl()).into(guidanceViewImage);
+    } else if (distanceTraveled + thresholdToShowGuidanceView >= currentStepTotalDistance) {
+      routeJunction = new RouteJunction(appendTokenTo(), currentStepTotalDistance);
+      new Picasso.Builder(getContext()).build().load(routeJunction.getGuidanceImageUrl()).into(guidanceViewImage);
+      animateShowGuidanceViewImage();
```

Comment:
is there a reason why the order of these two lines is reverse from https://github.com/mapbox/mapbox-navigation-android/pull/2519/files#diff-e993025c73d006405ac10453685867d0R389-R390 ?

Reply:
Nope. I was trying to see if it helps in animation.